tensorflow/lite/micro/testing/test_stm32f4_binary.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# Tests a 'stm32f4' STM32F4 ELF by parsing the log output of Renode emulation.
+#
+# First argument is the ELF location.
+# Second argument is a regular expression that's required to be in the output logs
+# for the test to pass.
+#
+# This script must be run from the top-level folder of the tensorflow github
+# repository as it mounts `pwd` to the renode docker image (via docker run -v)
+# and paths in the docker run command assume the entire tensorflow repo is mounted.
+
+declare -r ROOT_DIR=`pwd`
+declare -r TEST_TMPDIR=/tmp/test_stm32f4_binary/
+declare -r MICRO_LOG_PATH=${TEST_TMPDIR}
+declare -r MICRO_LOG_FILENAME=${MICRO_LOG_PATH}/logs.txt
+mkdir -p ${MICRO_LOG_PATH}
+
+docker build -t renode_stm32f4 \
```

Comment:
`tensorflow/lite/micro/tools/make/Makefile:250: warning: overriding recipe for target 'tensorflow/lite/micro/tools/make/downloads/cmsis'
tensorflow/lite/micro/tools/make/Makefile:250: warning: ignoring old recipe for target 'tensorflow/lite/micro/tools/make/downloads/cmsis'
tensorflow/lite/micro/testing/test_stm32f4_binary.sh tensorflow/lite/micro/tools/make/gen/stm32f4_cortex-m4/bin/hello_world_test '~~~ALL TESTS PASSED~~~'
tensorflow/lite/micro/testing/test_stm32f4_binary.sh: line 33: docker: command not found
tensorflow/lite/micro/examples/hello_world/Makefile.inc:33: recipe for target 'test_hello_world_test' failed
make: *** [test_hello_world_test] Error 127`

Reply:
I think that is related to hello_world_test binary is not building due to RAM being too small that I saw earlier. Although I did no longer see the error on the latest master branch. Increased RAM size anyway.